homeassistant/components/knx.py

```diff
@@ -74,7 +81,8 @@ def __init__(self, config):
 
         self.config = config
         self.should_poll = config.get("poll", True)
-        self._address = parse_group_address(config.get("address"))
```

Comment:
Address is q required parameter. I see that I did not check this. I will update this.